src/Filesystem/Driver/File.php

```diff
@@ -380,7 +382,7 @@ public function touch($path, $time = null): bool
      */
     public function filePutContents($path, $content, $mode = null)
     {
-        $result = @file_put_contents($path, $content, $mode);
+        $result = @file_put_contents($path, $content, $mode ?? 0);
```

Comment:
I would have just modified the default value for `$mode` on line 383, but this is fine